Pa3/Durchführung/Project/app/State/InitializeMcu.c

```diff
@@ -36,11 +40,28 @@
 
 EventEnum InitializeMcu_InitializeAll(void)
 {
-
+  //HAL
+  if (Gpio_init(void) != 0)
+  {
+    ErrorHandler_halt(ERRORHANDLER_STARTUP_INIT_FAIL);
+  }
+  Pwm_init(void);
+  Tick_Timer_init(void);
+  Irq_init(void);
```

Comment:
on function call the void is not necessary, only in declarations and definitions
```suggestion
  Pwm_init();
  Tick_Timer_init();
  Irq_init();
```